zerver/openapi/zulip.yaml

```diff
@@ -8450,6 +8450,19 @@ paths:
                           user automatically unmuting or following the topic in question.
 
                           **Changes**: New in Zulip 8.0 (feature level 218).
+                      message_url:
+                        type: string
+                        description: |
+                          The URL of the sent message.
```

Comment:
This also is not specified clearly; this is a URL for a link to that message within its conversation.